lib/ejs.js

```diff
@@ -641,7 +641,7 @@ Template.prototype = {
         e.message += ' while compiling ejs\n\n';
         e.message += 'If the above error is not helpful, you may want to try EJS-Lint:\n';
         e.message += 'https://github.com/RyanZim/EJS-Lint';
-        if (!e.async) {
+        if (!opts.async) {
```

Comment:
I&nbsp;already fixed&nbsp;this in&nbsp;https://github.com/mde/ejs/pull/460 (https://github.com/mde/ejs/commit/d6376fa71bd0f38f11a65d16df9c407bc61d6487).